nslsii/tests/test_xspress3.py

```diff
@@ -143,7 +143,7 @@ def test_instantiate_channel_class():
 
     assert channel_2.image.dtype_str == "uint32"
     assert channel_2.image.shape == (4096,)
-    assert channel_2.image.dims == ("bin_count",)
+    assert channel_2.image.dims == ('frame_number', 'number_channels', 'bin_count',)
```

Comment:
`black` prefers double quotes. Can we switch those?
```suggestion
    assert channel_2.image.dims == ("frame_number", "number_channels", "bin_count",)
```